lib/rule-tester/flat-rule-tester.js

```diff
@@ -84,13 +84,15 @@ const { ConfigArraySymbol } = require("@humanwhocodes/config-array");
  * testerDefaultConfig must not be modified as it allows to reset the tester to
  * the initial default configuration
  */
-const testerDefaultConfig = { rules: {} };
+const testerDefaultConfig = { files: ["**/*.*"], rules: {} };
```

Comment:
```suggestion
const testerDefaultConfig = { files: ["**"], rules: {} };
```
Maybe this way? This would match files without an extension, too.

Reply:
I am currently trying to find a pattern to allow the filename without an extension, but:
- "**/*": does not match
- "*" and "**" works but not when given a path (eg. `/path/to/file` regardless of existence of a file extension)

Reply:
Yeah, somehow "**" does match `path/to/file` but not `/path/to/file`.

Reply:
Paths should be relative, or inside `cwd` if absolute, so a generic absolute `/path/to/file` won't work regardless of the pattern.

I think `"**"` would do the work.

Reply:
Yes, you are correct. Changed the file pattern to `"**"` and added test cases.